submissions/morenko/task-frogger/js/app.js

```diff
@@ -0,0 +1,177 @@
+const countRows = 6;
+const countColumns = 5;
+
+const tileHight = 83;
+const tileWidth = 105;
+
+class Position {
+    startX;
+    startY;
+
+    x;
+    y;
+
+    minY;
+    minX;
+    maxY;
+    maxX;
+    
+    constructor( row, col ) {
+        
+        this.startX = (tileWidth * col) - (tileWidth);
+        this.startY = (tileHight * row) - (tileHight / 2);
+        this.x = this.startX;
+        this.y = this.startY;
+
+        this.maxY = tileHight * (countRows - 1);
+        this.maxX = tileWidth * (countColumns - 1);
+        this.minY = tileHight * (row - 1) - this.startY;
+        this.minX = 0;
+    }
+
+    reset() {
+        this.x = this.startX;
+        this.y = this.startY;
+        
+    }
+}
+
+class Enemy {
+
+    speed = (Math.floor(Math.random(4) * (500 - 100 + 1)) + 100);
+    sprite = 'images/Rock.png';
+
+    constructor ( PositionEnemy ) {
+        this.position = PositionEnemy;
+    }
+
+    update (dt) {
+        if (this.position.x <= this.position.maxX) {
+            this.position.run(this.speed, dt);
+        } else {
+            this.position.reset();
+        }
+    }
+
+    render = function() {
```

Comment:
There is some inconsistency in defining your methods. Some of them you go straign `update (dt) { ...`. And here you make `render = function() {...`. Why not `render () {...`?